pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -129,6 +129,27 @@ def feed_subscription_stats(self, network_name: str):
 
         return self._query_db(query)
 
+    @enforce_types
+    def feed_daily_subscriptions_by_feed_id(self, feed_id: str):
+        query = f"""
+        WITH date_counts AS (
+            SELECT
+                CAST(TO_TIMESTAMP(timestamp / 1000) AS DATE) AS day,
+                COUNT(*) AS count,
+                SUM(last_price_value) AS revenue
+            FROM
+                {Subscription.get_lake_table_name()}
+            WHERE
+                ID LIKE '%{feed_id}%'
+            GROUP BY
+                CAST(TO_TIMESTAMP(timestamp / 1000) AS DATE)
```

Comment:
we have to avoid from recalculation for grouping. we can use `day` instead of this line